app/controllers/blog_controller.rb

```diff
@@ -1,4 +1,3 @@
 class BlogController < ApplicationController
-  def index
-  end
+  def index; end
```

Comment:
- [ ] in your migration file, your users table according to the ERD supposed to have a name column with a string value but I think you create firstname and lastname instead . Please fix it because it is very important to follow the project requirement to make your app work as expected . :+1:

Reply:
ok.